universe/interface.go

```diff
@@ -265,6 +265,23 @@ type Registrar interface {
 		leaf *MintingLeaf) (*IssuanceProof, error)
 }
 
+// IssuanceItem is an item that can be used to register a new issuance within a
+// base universe.
+type IssuanceItem struct {
+	// ID is the identifier of the base universe that the item should be
+	// registered within.
+	ID Identifier
+
+	// Key is the base key that the leaf is or will be stored at.
+	Key BaseKey
+
+	// Leaf is the minting leaf that was created.
+	Leaf *MintingLeaf
+
+	// MetaReveal is the meta reveal that was created.
+	MetaReveal *proof.MetaReveal
```

Comment:
Still here in latest diff.

Reply:
It is being used [here](https://github.com/lightninglabs/taproot-assets/blob/082a08b6a04ffc58bc46f94aec817648f884031a/universe/base.go#L276). It just wasn't set for the call to `verifyIssuanceProof` where I unrolled the parameters again.